src/app/profile/update/update.component.html

```diff
@@ -153,7 +153,25 @@ <h3 class="col-sm-12 profile-wrapper-username">{{context?.user?.attributes?.full
             <span class="fields-status-pf">characters remaining</span>
           </span>
         </div>
-        
+
+        <div class="form-group padding-top-40">
+          <div class="padding-top-10">
+            <h2>Experimental Features</h2>
+          </div>
+          <div>
+            <h5>The legal text goes here. Please don't blame us for using this feature.</h5>
+          </div>
+          <div class="col-sm-1 col-lg-1 padding-left-0 padding-right-0">
+            <input type="checkbox" class="form-control" id="experimental" name="experimental" #_experimental
```

Comment:
We probably don't need the #_experimental here unless you need to get a reference to this element in the component code.